wrapper/Tools/OpenMMMD.py

```diff
@@ -703,6 +703,8 @@ def setupForcefields(system, space):
 
         for molnum in molnums:
             mol = molecules.molecule(molnum)[0].molecule()
+            if not mol.hasProperty("restrainedAtoms"):
```

Comment:
I assume the property is case sensitive, no? Here you use `restrainedAtoms`, whereas below you use `restrainedatoms`.